language-server/vscoqtop/lspManager.ml

```diff
@@ -43,6 +43,8 @@ let full_messages = ref false
 
 let block_on_first_error = ref true
 
+let interp_to_next_position = ref true
```

Comment:
Instead of making this an option in the backend, I would rather keep the option in the frontend and make a new lsp request called ```vscoq/interpretToNextPoint```. This would need to be implemented in the extProtocol.ml file.
Then depending on the option in the frontend, if the user launches a ```Coq: Interpret To Point``` command it either launches a ```vscoq/interpretToPoint``` or ```vscoq/interpretToNextPoint```:
```typescript
export const sendInterpretToPoint = (editor: TextEditor,  client: Client) => {
    const textDocument = makeVersionedDocumentId(editor);
    const position = editor.selection.active;
    const mode = workspace.getConfiguration("vscoq.proof").pointInterpretationMode;
    if(mode === 0) {
        client.sendNotification("vscoq/interpretToPoint", {textDocument: textDocument, position: position});
    } else {
        client.sendNotification("vscoq/interpretToNextPoint", {textDocument: textDocument, position: position});
    }
 };
```

```